chag

```diff
@@ -173,7 +213,7 @@ latest() {
 
 # Get the line number of a specific changelog entry.
 find_entry() {
-  result=$(entries_and_lines | grep -E ":${1//./\\.}$" | cut -d ':' -f 1)
+  result=$(entries_and_lines | grep "${1//./\\.}" | cut -d ':' -f 1)
```

Comment:
Mine didn't work after that line was changed to the new one. But I should try it again.